update_translations.py

```diff
@@ -0,0 +1,116 @@
+import subprocess
```

Comment:
Any thoughts on using something like GitPython instead of making a new subprocess for every git command?